### PR TITLE
[fix][master]: fix condition npe

### LINKED ISSUE #10476: [Bug] [Master] workflow instance never stop because conditional task has no downstream task
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

If condition have no downstream task, master will npe during workflow run.And workflow never stop.
image
image

### What you expected to happen

The workflow instance completed. 

### How to reproduce

image
image
1.create a condition task that have no downstream task.
2.run workflow.

### Anything else

etc.

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by huangchenguang123**
close #10476

### CODE REVIEW

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/DagHelper.java [hunk 1]
```diff
@@ -32,13 +32,7 @@
...
 import org.apache.commons.collections.CollectionUtils;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```
Comment (SbloodyS) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/DagHelper.java]: Please avoid import *

Comment (huangchenguang123) [reply dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/DagHelper.java]: IDEA make import * when i impot, I had fix it.

**Review by SbloodyS — APPROVED** [review]
LGTM. Thanks.

**Comment by SbloodyS** [comment]
Hi @huangchenguang123 , please merge the latest dev branch since we have fixed the flaky ut error.

**Pull request merged**